setup.py

```diff
@@ -82,6 +82,7 @@ def run(self):
     "shtab>=1.3.0,<2",
     "rich>=3.0.5",
     "dictdiffer>=0.8.1",
+    "wheel>=0.35.1",
```

Comment:
`wheel` should never be a _runtime_/_install_time_ dependency, it's a _buildtime_ dependency. It's your responsibility to install wheels (I know that this isn't ideal, python packaging sucks). We do that for `dvc[tests]` install already.

I am against this change. @DDR0, please see if you had an older version of the "wheel" installed already (that could be the reason for "invalid command"). If not, please see if adding to `setup_requires` work (this is deprecated AFAIK).

Reply:
Good catch! Didn't notice that this wasn't in `[tests]`. Also ran into this build error today in #4607 .

Reply:
We could consider installing it explicitly in `scripts/build_package.sh`.